src/Types.hs

```diff
@@ -27,38 +27,63 @@ import qualified Data.Csv as CSV
 
 -- | Globally Unique ID (for reference in the database).  (FIXME: should we have different id types
 -- for different object types?)
-type GUID = Integer
+newtype GUID a = GUID Integer
```

Comment:
Bikeshedding: I think we should call this something other than GUID, since [GUID has a specific meaning](https://en.wikipedia.org/wiki/Globally_unique_identifier).